core/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -193,7 +196,8 @@ private TachyonFS(TachyonConf tachyonConf) throws IOException {
   /**
    * Update the latest block access time on the worker.
    * 
-   * @param blockId the local block's id
+   * @param blockId
+   *          the local block's id
```

Comment:
This is just Eclipse auto-formatting applied using the Tachyon Eclipse settings file